bazel_codegen/lib/src/summaries/resolvers.dart

```diff
@@ -57,51 +59,49 @@ class SummaryResolvers implements Resolvers {
   }
 }
 
-/// a [Resolver] backed by an [AnalysisContext].
+/// a [Resolver] backed by an [AnalysisDriver].
 class AnalysisResolver implements ReleasableResolver {
-  final AnalysisContext _analysisContext;
+  final AnalysisDriver _analysisDriver;
   final List<AssetId> _assetIds;
 
-  AnalysisResolver(this._analysisContext, this._assetIds);
+  AnalysisResolver(this._analysisDriver, this._assetIds);
 
   @override
```

Comment:
done, I always want to put it below but the formatter really doesn't like that haha